.github/files/update-jetpack-staging-sites.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+# This script is ran as part of the `UpdateJetpackStaging` TeamCity build.
+# It updates pre-defined Atomic test sites with the latest plugin versions.
+
+####################################################
+## Script variables.
+####################################################
+
+TMP_DIR=$(mktemp -d) # Temp dir where the plugin .zip files are downloaded and unpacked.
+REMOTE_DIR='/srv/htdocs/jetpack-staging' # Remote dir where the unpacked plugin files are synced to.
+PLUGINS=( "jetpack" "jetpack-mu-wpcom-plugin" ); # Plugins to update.
+declare -A PLUGIN_VERSIONS # Array used to hold fetched plugin versions.
+declare -A PLUGIN_DOWNLOAD_URLS # Array used to hold fetched plugin download URLs.
+
+SITES='{
```

Comment:
I note most of the data in here isn't actually used. You could probably just do
```
SITES=(
    jetpackedge
    jetpackedgephp74
    etc
)
```
and then below
```
for host in "${SITES[@]}"; do
    key="${host}.wpcomstaging.com"
    ssh_string="${host}.wordpress.com@sftp.wp.com"

    ....
done
```
(or even skip the extra variables and just inline them where used).

Reply:
Yeah it could be cleaned up a bit, though I personally like having more of that site info directly in the file.